File: app/src/main/java/info/nukoneko/cuc/android/kidspos/entity/Sale.java
package info.nukoneko.cuc.android.kidspos.entity;

import java.util.Date;

public final class Sale {
    private int id;
    private String barcode;
    private Date createdAt;
    private int points;
    private int price;
    private String items;
    private int storeId;
    private int staffId;

    public int getId() {
        return id;
    }

    public String getBarcode() {
        return barcode;
    }

    public Date getCreatedAt() {
        return createdAt;
    }

    public int getPoints() {
        return points;
    }

    public int getPrice() {
        return price;
    }

    public String getItems() {
        return items;
    }

    public int getStoreId() {
        return storeId;
    }

    public int getStaffId() {
        return staffId;
    }
}

File: app/src/main/java/info/nukoneko/cuc/android/kidspos/entity/Sale.kt
package info.nukoneko.cuc.android.kidspos.entity

import java.util.Date

data class Sale(val id: Int, val barcode: String,
           val createdAt: Date,
           val points: Int, val price: Int,
           val items: String, val storeId: Int, val staffId: Int)
